server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -259,7 +250,9 @@ public String executor() {
                     filteredHeaders,
                     contentType,
                     content,
-                    requestIssuerIdentity,
+                    identityService.getTokenManager()
+                        .issueOnBehalfOfToken(Map.of("aud", discoveryExtensionNode.getId()))
```

Comment:
This looks fine to me, but we may want to leave a note that `discoveryExtensionNode.getId()` is what is being used to get the extension's unique id. It looks like extensions only support a single extension node atm and that node is assigned the unique id of the extension, but that would probably be subject to change in the future.

I don't see any dependencies that have the constant "aud", but it could make sense to extract this to an enum like `JwtStandardClaims`.

Reply:
Sounds good